src/main/java/com/google/sps/servlets/RepresentativeListServlet.java

```diff
@@ -34,7 +34,8 @@
     private final String apiKey;
 
     public RepresentativeListServlet() {
-        apiKey = System.getenv(Constants.CIVIC_API_KEY);
+        // apiKey = System.getenv(Constants.CIVIC_API_KEY);
+        apiKey = "AIzaSyAHQRFZpZBZSOeUI-4pmfoHV4SM0eBSlS4";
```

Comment:
shouldn't be here, add to git ignore

Reply:
use the environment. ask nikita for help if you need.